xonsh/ptk/shortcuts.py

```diff
@@ -61,6 +64,8 @@ def prompt(self, message='', **kwargs):
 
         # Create CommandLineInterface.
         if self.cli is None:
+            if self.major_minor <= (0, 57):
```

Comment:
Hey @scopatz -- `prompt_toolkit` master still reports itself as version 0.57 so the `get_rprompt_tokens` kwarg never makes it past this conditional.  
I removed it and then `$RIGHT_PROMPT` works as expected.